content/_index.md

```diff
@@ -46,3 +46,78 @@ body = '''
 PyBaMM can be combined with any of the advanced data science and machine learning tools in Python's extensive ecosystem.
 '''
 {{< /grid >}}
+
+<div align="center">
+<h2>Trusted by top battery researchers and engineers</h2>
+</div>
+
+<div style="display:flex; justify-content:space-around; align-items:center; flex-wrap:wrap">
```

Comment:
```suggestion
<div style="display:flex; justify-content:space-around; align-items:center; flex-wrap:wrap; background:white; border-radius:0.5em 0.5em 0 0">
```

Something like this should fix the backgrounds so that this gallery has a white one. This should also wrap them in a slightly rounded border

Reply:
What's weird is that the images themselves have transparent backgrounds, so they should be fine on a black background
![Screenshot 2024-07-19 at 8 57 58 AM](https://github.com/user-attachments/assets/f116a861-1fb7-48fb-9958-143c592eafad)

Reply:
We could also remove the light/dark mode and always use light mode

Reply:
For light mode, I see that NumPy permanently sets it to white, while SciPy does not. I'll try to find out how.